tests/framework/filters/AccessRuleTest.php

```diff
@@ -39,14 +40,19 @@ protected function mockRequest($method = 'GET')
     }
 
     /**
+     * @param string optional user id
      * @return User
      */
-    protected function mockUser()
+    protected function mockUser($userid = null)
```

Comment:
It is used by the already existing test cases (testMatchAction, testMatchVerb). I didn't want to break those, neither changes tests I'm not working with.
See lines: 121 and 185